rotkehlchen/chain/aggregator.py

```diff
@@ -972,25 +980,30 @@ def query_eth_balances(
         self.query_evm_chain_balances(chain=SupportedBlockchain.ETHEREUM)
         self.query_defi_balances()
         self._add_eth_protocol_balances(eth_balances=self.balances.eth)
-        self._add_staking_protocol_balances()
+        self._add_staking_protocol_balances(chain_id=ChainID.ETHEREUM)
 
-    def _add_staking_protocol_balances(self) -> None:
+    def _add_staking_protocol_balances(self, chain_id: CHAIN_IDS_WITH_BALANCE_PROTOCOLS) -> None:  # noqa: E501
         """
         Query protocols that have staked balances and special contracts need to be called.
         This includes:
         - Curve gauges
         - Convex gauges
         """
-        balance_inquirers = [CurveBalances, ConvexBalances]
-        for inquirer_cls in balance_inquirers:
-            inquirer: ProtocolWithBalance = inquirer_cls(
+        chain = ChainID.to_blockchain(chain_id)
+        inquirer = self.get_chain_manager(chain).node_inquirer  # type: ignore  # chain's type here is a subset of the type expected by get_chain_manager  # noqa: E501
+        existing_balances = self.balances.get(chain)
+        protocols = CHAIN_TO_BALANCE_PROTOCOLS[chain_id]
+        for protocol in protocols:
```

Comment:
Please as I have said many times don't make variables that are just used in one place

```suggestion
        for protocol in CHAIN_TO_BALANCE_PROTOCOLS[chain_id]:
```